indico/modules/rb/controllers/user/reservations.py

```diff
@@ -24,6 +24,7 @@
 
 from indico.core.db import db
 from indico.core.errors import IndicoError, NoReportError, NotFoundError
+from indico.modules.rb import settings as rb_settings
```

Comment:
It seems like a legitimate choice. `indico.modules.rb.settings` should already give you some context on where the settings belong. I don't see any problem with that pattern, even if it's true that at some point we decided to do it otherwise.